efopen/ef_instanceinit.py

```diff
@@ -175,6 +206,7 @@ def main():
   log_info("platform: {} service: {}".format(WHERE, service))
 
   merge_files("all")
+  merge_files("ssh", True)
```

Comment:
Would use the named argument here instead of a plain `True`.
```suggestion
  merge_files("ssh", skip_on_user_group_error=True)
```

Other than that, 👍